src/value/owned.rs

```diff
@@ -666,20 +719,20 @@ mod value_tests {
     #[rstest(
         elem, ion_type, valid_ops_iter, op_assert,
         case::null(
-            IonType::Null.into(),
+            OwnedElement::new_null(IonType::Null),
             IonType::Null,
             IsNull,
             &|e: &OwnedElement| assert_eq!(true, e.is_null())
         ),
         // TODO more null testing (probably its own fixture)
```

Comment:
Do these tests matter now that you have generic ones?

Reply:
I meant to do this in separate PR as it would require to add more tests in generic test suite.